shub/image/init.py

```diff
@@ -70,26 +68,37 @@ def list_recommended_python_reqs(ctx, param, value):
     ctx.exit()
 
 
+def _deprecate_base_deps_parameter(ctx, param, value):
+    if value:
+        click.echo("WARNING: --base-deps parameter is deprecated. "
+                   "Please use --add-deps parameter instead.",
+                   err=True)
+    return value
+
+
 @click.command(help=HELP, short_help=SHORT_HELP)
 @click.option("--list-recommended-reqs", is_flag=True, is_eager=True,
               expose_value=False, callback=list_recommended_python_reqs,
               help="list recommended python requirements")
 @click.option("--project", default="default",
               help="project name to get settings module from scrapy.cfg")
-@click.option("--base-image", default="python:2.7",
+@click.option("--base-image", default="scrapinghub/scrapinghub-stack-scrapy:1.3",
```

Comment:
I'd probably move the base image to some constant `DEFAULT_BASE_IMAGE` on top of the module, but that's a minor thing and can be done later.